src/main/resources/transports/boats.tsv

```diff
@@ -78,8 +78,8 @@
 # Fossil Island, Underwater								
 3724 3807 0	3734 3893 0	Travel Rowboat 30914					4	
 3734 3893 0	3724 3807 0	Travel Rowboat 30915					3	
-3734 3893 0	3768 3898 0	Travel Rowboat 30915					3	
-3768 3898 0	3734 3893 0	Travel Rowboat 30919					3	
+3734 3893 0	3764 3899 0	Travel Rowboat 30915					3	
+3763 3899 0	3733 3893 0	Travel Rowboat 30919					3	
```

Comment:
I don't think these coordinates are wrong. The new coordinates you have added are in the ocean
![image](https://github.com/user-attachments/assets/e7f4dd54-5dc3-4bd0-8962-5c9964f025b8)

Reply:
Looks like there's something wrong with the map in this area?

<img width="1258" height="753" alt="{5498EF78-63D6-4E42-878E-F90A600472D5}" src="https://github.com/user-attachments/assets/c5d04b2c-96d0-46fc-8c79-5bbf70112089" />

Reply:
Oh, nevermind, I know what it is. They probably moved the isle slightly in this update to align it with the zipline. Means you'll need to update the collision map too probably.

Reply:
Ah, good observation. I will update the collision map eventually